WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -123,6 +136,92 @@ private extension ProductCreationAIStartingInfoView {
         .disabled(viewModel.features.isEmpty)
     }
 }
+
+private extension ProductCreationAIStartingInfoView {
+    struct PackagePhotoView: View {
+        @ScaledMetric private var scale: CGFloat = 1.0
+
+        let imageState: EditableImageViewState
+
+        let onTapViewPhoto: () -> Void
+        let onTapReplacePhoto: () -> Void
+        let onTapRemovePhoto: () -> Void
+
+        var body: some View {
+            HStack(alignment: .center, spacing: Layout.spacing) {
+                EditableImageView(imageState: imageState,
+                                  emptyContent: {})
+                .frame(width: Layout.packagePhotoSize * scale, height: Layout.packagePhotoSize * scale)
+                .cornerRadius(Layout.cornerRadius)
+
+                Text(Localization.photoUploaded)
+                    .bodyStyle()
+
+                Spacer()
+
+                Menu {
+                    Button(Localization.viewPhoto) {
+                        onTapViewPhoto()
+                    }
+                    Button(Localization.replacePhoto) {
+                        onTapReplacePhoto()
+                    }
+                    Button(role: .destructive) {
+                        onTapRemovePhoto()
+                    } label: {
+                        Text(Localization.removePhoto)
+                    }
+                } label: {
+                    Image(systemName: "ellipsis")
+                        .frame(width: Layout.ellipisButtonSize * scale, height: Layout.ellipisButtonSize * scale)
+                        .bodyStyle()
+                        .foregroundStyle(Color.secondary)
+                }
+            }
+            .padding(Layout.padding)
+            .background(Color(.systemColor(.systemGray6)))
+            .clipShape(
+                .rect(
+                    bottomLeadingRadius: Layout.textFieldOverlayCornerRadius,
+                    bottomTrailingRadius: Layout.textFieldOverlayCornerRadius
+                )
+            )
+        }
+
+        enum Layout {
+            static let spacing: CGFloat = 16
+            static let cornerRadius: CGFloat = 4
+            static let padding = EdgeInsets(top: 12, leading: 16, bottom: 12, trailing: 16)
+            static let textFieldOverlayCornerRadius: CGFloat = 8
+            static let packagePhotoSize: CGFloat = 48
+            static let ellipisButtonSize: CGFloat = 24
+        }
+
+        enum Localization {
+            static let photoUploaded = NSLocalizedString(
+                "productCreationAIStartingInfoView.packagePhotoView.viewPhoto",
+                value: "Photo uploaded",
```

Comment:
Nit: Should we use "Photo selected" instead? "uploaded" sounds like the photo has been uploaded somewhere - it's misleading to me.

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/13193/commits/a995988b6d03af1838010be175b9a38930eae9a4